wolfcrypt/src/logging.c

```diff
@@ -24,6 +24,10 @@
     #include <config.h>
 #endif
 
+#ifndef WOLFSSL_USER_SETTINGS
+    #include <wolfssl/options.h>
```

Comment:
options.h can never go in the library proper.  It's a user file, no a library build file.

Reply:
Got it.  I'll refactor the way autotools communicates the configure args and CFLAGS to logging.c, and I think `HAVE_WC_INTROSPECTION` can move to config.h.  Details TBD.

Btw, when I first set this mechanism up, I had autotools just pass giant `-D` strings to the compiler, but I was unhappy with the massive increase in noise with `make V=1`.  I'm still intent on avoiding that.

Reply:
This turned out to be easy to fix.  Phew.